### PR TITLE
Pre spawned pills, now show amount of reagent

### PR DESCRIPTION
**Pull request by NortonDK**
This pull will make all the pre spawned pills around the station show how much of the reagent is in the pill. just like when the chemist makes em.

(now it should be done correctly and all that :D)

### CODE REVIEW

**Comment by NortonDK** [comment]
!review

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -81,63 +81,63 @@
 
 //Pills
 /obj/item/weapon/reagent_containers/pill/antitox
-	name = "Anti-toxins pill"
+	name = "Anti-toxins pill (25u)"
```
Comment (Pacmandevil) [inline code/modules/reagents/reagent_containers/pill.dm]: This is a really iffy way of doing it imo, a better way would be to add onto the name in Initialize() with the total num of reagents in it.

Comment (Pacmandevil) [reply code/modules/reagents/reagent_containers/pill.dm]: Which shouldn't be a manual entry! That'd just lead to people forgetting to update it in the off chance it's changed.

I think there's a proc to get the total reagents in any specific object.

Comment (NortonDK) [reply code/modules/reagents/reagent_containers/pill.dm]: As the newbie i am, where could i figure out where such proc exist?

Comment (Pacmandevil) [reply code/modules/reagents/reagent_containers/pill.dm]: You'd want to check
/obj/item/weapon/reagent_containers/pill/Initialize()

You'd probably run into the issue of it Initializing before reagents are added, however. Easiest/Shittiest way would be throwing a timer in the Initialize, but there's probably a better way to do it.

I'd suggest asking around the Code dungeon and finding someone who knows what they're doing.

**Comment by Alberyk** [comment]
It looks ugly in the name, move it to the description or an examine proc.

**Comment by NortonDK** [comment]
Alberyk its the same naming that is defualt on the pill maker

**Comment by HeyBanditoz** [comment]
It really doesn't. It's an awful solution. Just do what @Pacmandevil said.

**Pull request closed without merging**